PythonRegression/tests/features/steps/api_test_steps.py

```diff
@@ -38,7 +38,7 @@
         responseList:   Same as responseValue, ecept it places the results into a list
         bool:           Bool argument, returns True or False
  
-'''
+"""
```

Comment:
hmm
question, shouldn't this docstring be under the method as described in PEP 257?